src/drivers/angular2.js

```diff
@@ -0,0 +1,99 @@
+import { omit, replaceObject } from '../lib';
+/* eslint-disable new-cap, object-shorthand */
+
+
+export let angular2 = {
+
+    global() {
```

Comment:
Yes very annoying.

Import * as core from '@angular/core'
That would solve big part of the problem however will be annoying to angular users used to import core functions individually. Your thoughts?